test/pkg_autocorrects_test.cc

```diff
@@ -90,140 +90,6 @@ struct TestPackageFile {
-
-    auto test = TestPackageFile::create(gs, "my_package/__package.rb", pkg_source);
-
-    auto &package = test.targetPackage(gs);
-    ENFORCE(package.exists());
-    auto addImport = package.addImport(gs, test.newPackage(gs), false);
-    ENFORCE(addImport, "Expected to get an autocorrect from `addImport`");
-    auto replaced = addImport->applySingleEditForTesting(pkg_source);
-    CHECK_EQ(expected, replaced);
-}
-
-TEST_CASE("Add test import with only existing exports") {
-    core::GlobalState gs(errorQueue);
-    gs.initEmpty();
-
-    string pkg_source = "class Opus::MyPackage < PackageSpec\n"
-                        "  export Opus::SomethingElse\n"
-                        "end\n";
-
-    string expected = "class Opus::MyPackage < PackageSpec\n"
-                      "  test_import Opus::ExamplePackage\n"
-                      "  export Opus::SomethingElse\n"
-                      "end\n";
-
-    auto test = TestPackageFile::create(gs, "my_package/__package.rb", pkg_source);
-
-    auto &package = test.targetPackage(gs);
-    ENFORCE(package.exists());
-    auto addImport = package.addImport(gs, test.newPackage(gs), true);
-    ENFORCE(addImport, "Expected to get an autocorrect from `addImport`");
-    auto replaced = addImport->applySingleEditForTesting(pkg_source);
-    CHECK_EQ(expected, replaced);
-}
-
-TEST_CASE("Add import to package with neither imports nor exports") {
-    core::GlobalState gs(errorQueue);
-    gs.initEmpty();
-
-    string pkg_source = "class Opus::MyPackage < PackageSpec\n"
-                        "end\n";
-
-    string expected = "class Opus::MyPackage < PackageSpec\n"
-                      "  import Opus::ExamplePackage\n"
-                      "end\n";
-
-    auto test = TestPackageFile::create(gs, "my_package/__package.rb", pkg_source);
-
-    auto &package = test.targetPackage(gs);
-    ENFORCE(package.exists());
-    auto addImport = package.addImport(gs, test.newPackage(gs), false);
-    ENFORCE(addImport, "Expected to get an autocorrect from `addImport`");
-    auto replaced = addImport->applySingleEditForTesting(pkg_source);
-    CHECK_EQ(expected, replaced);
-}
-
-TEST_CASE("Add test import to package with neither imports nor exports") {
-    core::GlobalState gs(errorQueue);
-    gs.initEmpty();
-
-    string pkg_source = "class Opus::MyPackage < PackageSpec\n"
-                        "end\n";
-
-    string expected = "class Opus::MyPackage < PackageSpec\n"
-                      "  test_import Opus::ExamplePackage\n"
-                      "end\n";
-
-    auto test = TestPackageFile::create(gs, "my_package/__package.rb", pkg_source);
-
-    auto &package = test.targetPackage(gs);
-    ENFORCE(package.exists());
-    auto addImport = package.addImport(gs, test.newPackage(gs), true);
-    ENFORCE(addImport, "Expected to get an autocorrect from `addImport`");
-    auto replaced = addImport->applySingleEditForTesting(pkg_source);
-    CHECK_EQ(expected, replaced);
-}
-
```

Comment:
Why do we have to delete all these tests? Are they covered by some other aspect of our tests?